katcp/test/test_inspecting_client.py

```diff
@@ -155,6 +157,88 @@ def test_update_index(self):
         self.assertTrue(index['B'].get('_changed'))
 
 
+class TestInspectingClientInspect(tornado.testing.AsyncTestCase):
+    """Check that inspection populates the request/sensor index correctly"""
+
+    def setUp(self):
+        super(TestInspectingClientInspect, self).setUp()
+        self.server = DeviceTestServer('', 0)
+        start_thread_with_cleanup(self, self.server, start_timeout=1)
+        self.server_with_timeout_hints = DeviceTestServerWithTimeoutHints('', 0)
+        start_thread_with_cleanup(self, self.server_with_timeout_hints,
```

Comment:
You don't seem to be using the two servers instantiated here.  The tests use instances created by `_get_server`.